{"name": "Изменение email, на уже существующие в системе у авторизованного пользователя", "status": "passed", "start": 1729621191071, "stop": 1729621191593, "uuid": "526a59f5-8563-4e86-a381-a15adb2174d1", "historyId": "e9b65f5e1545c5c6e606b37e3557a81c", "testCaseId": "e9b65f5e1545c5c6e606b37e3557a81c", "fullName": "tests.test_change_data_user.TestChangeDataUser#test_changes_email_error_403", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_data_user"}, {"name": "subSuite", "value": "TestChangeDataUser"}, {"name": "host", "value": "User-PC"}, {"name": "thread", "value": "8308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_data_user"}]}